llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -619,6 +622,13 @@ static void computeStats(CASDB &CAS, ArrayRef<CASID> TopLevels) {
   Collector.printToOuts(TopLevels);
 }
 
+static raw_fd_ostream &outCSV() {
+  std::error_code EC;
+  static raw_fd_ostream S(ComputeStatsCSV, EC, sys::fs::OF_None);
+  assert(!EC);
```

Comment:
I will make a proper `llvm::Error` here and `ExitOnError` with proper error message. This should not be an assert since this is a user input error.